common/src/main/java/com/unrealdinnerbone/jamd/util/TelerportUtils.java

```diff
@@ -43,7 +43,7 @@ private static Optional<BlockPos> findPortalLocation(Level worldTo, BlockPos fro
             return Optional.of(fromPos.above());
         }
 
-        int range = 5;
+        int range = 3;
```

Comment:
Cold you explain the range change here?

Reply:
Just tightening the range so the function doesn't have to search as far.